at-luantran/Week4/Human_Resources/models/employee.rb

```diff
@@ -0,0 +1,35 @@
+require 'bcrypt'
+require 'mysql2'
+
+class Employee
+  attr_accessor :id, :name, :team, :role, :username, :password, :leave_balance
+  @connect = Mysql2::Client.new(host: 'localhost', username: 'root',
+                                password: 'hoichuphong513',
+                                database: 'LuanTranHumanResources')
+  def initialize; end
+
+  def self.check_login(username, password)
+    sql = "SELECT * FROM `LuanTranHumanResources`.`employee` WHERE username = '#{username}'"
+    result = @connect.query(sql)
+    result.each do |x|
+      BCrypt::Password.new(x.values_at('password').first) == password
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/03700b46d0ae03fcf2370fdd1cf42e5797805a69/at-luantran/Week4/Human_Resources/models/employee.rb#L15"></span>Lint/Void: Operator <code>==</code> used in void context.</td>
    </tr>
  </tbody>
</table>